railties/lib/rails/application/configuration.rb

```diff
@@ -34,6 +34,7 @@ def initialize(*)
         @filter_redirect                         = []
         @helpers_paths                           = []
         @hosts                                   = Array(([".localhost", IPAddr.new("0.0.0.0/0"), IPAddr.new("::/0")] if Rails.env.development?))
+        @hosts.concat(ENV["RAILS_DEVELOPMENT_HOSTS"].to_s.split(",")) if Rails.env.development?
```

Comment:
If someone takes "comma-separated hosts" to mean "foo, bar" rather than "foo,bar", so we'd add " bar" as a host, will it still work as expected? If not, maybe the docs could be more explicit or the parsing could be more forgiving.

Reply:
Good catch! I've added logic to remove whitespace